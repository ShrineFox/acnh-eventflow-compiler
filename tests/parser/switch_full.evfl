flow Test():
    switch TestActor1.Query0():
        default:
            TestActor1.Action0()
        case 2, 1:
            TestActor2.Action0()
