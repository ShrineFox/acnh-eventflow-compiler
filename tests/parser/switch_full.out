RootNode[name=Test, vardefs=[], out_edges=[Event6]]
ActionNode[name=Event0, action=EventFlowActionAction0(), params={}, out_edges=[]]
ActionNode[name=Event3, action=EventFlowActionAction0(), params={}, out_edges=[]]
SwitchNode[name=Event6, query=EventFlowQueryQuery0(), params={}, cases={'Event3': [2, 1], 'Event0': [0]}, out_edges=[Event3, Event0]]

